README.md

```diff
@@ -84,6 +84,11 @@ grunt.initConfig({
 });
 ```
 
+You can also inspect [`jsx` files used in React.js](http://facebook.github.io/react/docs/jsx-in-depth.html),
+simply by using that suffix in the `src` list.
+The command line version of `jsinspect` uses a specific flag for this,
+[namely `--jsx`](https://github.com/danielstjules/jsinspect#usage).
+
```

Comment:
I'm not sure I understand this completely.

I have to set files with `.jsx` plus enabling the flag via config?